docu/Concepts/TechnicalRequirements/BusinessEventProtocol.md

```diff
@@ -6,29 +6,30 @@ With the business event protocol the gradido application will capture and persis
-| RegisterEmailKlickTippEvent | 25    | the system registers the confirmed email at klicktipp                                                |
-| LoginEvent                  | 30    | the user presses the login button                                                                    |
-| RedeemLoginEvent            | 31    | the user presses the login button initiated by the redeem link                                       |
-| ActivateAccountEvent        | 32    | the system activates the users account during the first login process                                |
-| PasswordChangeEvent         | 33    | the user changes his password                                                                        |
-| TxSendEvent                 | 40    | the user creates a transaction and sends it online                                                   |
-| TxSendRedeemEvent           | 41    | the user creates a transaction and sends it per redeem link                                          |
-| TxRepeateRedeemEvent        | 42    | the user recreates a redeem link of a still open transaction                                         |
-| TxCreationEvent             | 50    | the user receives a creation transaction for his confirmed contribution                              |
-| TxReceiveEvent              | 51    | the user receives a transaction from an other user and posts the amount on his account               |
-| TxReceiveRedeemEvent        | 52    | the user activates the redeem link and receives the transaction and posts the amount on his account  |
-| ContribCreateEvent          | 60    | the user enters his contribution and asks for confirmation                                           |
-| ContribConfirmEvent         | 61    | the user confirms a contribution of an other user (for future multi confirmation from several users) |
-|                             |       |                                                                                                      |
+| EventType                           | Value | Description                                                                                            |
+| ----------------------------------- | ----- | ------------------------------------------------------------------------------------------------------ |
+| BasicEvent                          | 0     | the basic event is the root of all further extending event types                                       |
+| VisitGradidoEvent                   | 10    | if a user visits a gradido page without login or register                                              |
+| RegisterEvent                       | 20    | the user presses the register button                                                                   |
+| RedeemRegisterEvent                 | 21    | the user presses the register button initiated by the redeem link                                      |
```

Comment:
rights.REGISTER && args.redeemCode

do we want to have transaction link and contribution link logged differently?